composer.json

```diff
@@ -30,8 +30,7 @@
     "require-dev": {
         "symfony/process": "^4.4 || ^5.0",
         "phpunit/phpunit": "^8.5 || ^9.0",
-        "herrera-io/box": "~1.6.1",
-        "container-interop/container-interop": "^1.2"
+        "psr/container-implementation": "^1.0"
```

Comment:
that does not bring any benefit, as behat does not ask to choose an arbitrary implementation (and that's even more useless to require the virtual package as a dev dependency as the behat composer.json has to fulfill it anyway)